src/main/java/com/wavefront/sdk/entities/tracing/sampling/MinConditionalSampler.java

```diff
@@ -0,0 +1,28 @@
+package com.wavefront.sdk.entities.tracing.sampling;
+
+import java.util.List;
+
+/**
+ * This an extension of {@link CompositeSampler} that runs a pre-check to determine if a span is
+ * eligible for sampling. This implementation needs span to meet the minimum duration threshold
+ * to be considered for sampling, otherwise span is discarded.
+ *
+ * @author Anil Kodali (akodali@vmware.com)
+ */
+public class MinConditionalSampler extends CompositeSampler{
+
+  private final long minimumDuration;
```

Comment:
duration should always have a unit
rename this to minimumDurationMillis

Reply:
In hindsight, we should have renamed this variable to durationMillis
https://github.com/wavefrontHQ/wavefront-sdk-java/blob/master/src/main/java/com/wavefront/sdk/entities/tracing/sampling/DurationSampler.java#L10